### PR TITLE
Report cmake version on CI run

### PR DESCRIPTION
**Pull request by a4z**
To see what CMake the CI uses

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -121,8 +133,10 @@ jobs:
         $VERSION = 'v' + [regex]::Match((Get-Content .tool-versions), '^djinni (\d+.\d+.\d+)').captures.groups[1].value
         $URL = 'https://github.com/cross-language-cpp/djinni-generator/releases/download/' + $VERSION + '/djinni.bat'
         Invoke-WebRequest -Uri $URL -OutFile djinni.bat
+    - name: Report cmake version
+      run: cmake --version        
     - name: Configure cmake
-      run: cmake -S . -B build -DDJINNI_WITH_CPPCLI=ON -DCMAKE_INSTALL_PREFIX=build/check_install_root -DDJINNI_EXECUTABLE="$(((Get-Location).Path) -replace "\\","/")/djinni.bat" -G "Visual Studio 16 2019"
+      run: cmake -S . -B b:wuild -DDJINNI_WITH_CPPCLI=ON -DCMAKE_INSTALL_PREFIX=build/check_install_root -DDJINNI_EXECUTABLE="$(((Get-Location).Path) -replace "\\","/")/djinni.bat" -G "Visual Studio 16 2019"
```
Comment (jothepro) [inline .github/workflows/build.yml]: is this change intentional? 🤔

Comment (a4z) [reply .github/workflows/build.yml]: ups, where does that come from ? .....

ah .... one of the rare cases I used vim ... `:w` 
no, that should not be there , it was ment for vim, save a file ....
